meetings/2020-10-29.md

```diff
@@ -0,0 +1,86 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2020-10-29
+
+## Links
+
+* **Recording**: https://youtu.be/tKQRw8fAClo  
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/942
+
+## Present
+
+* Beth Griggs @BethGriggs (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Gabriel Schulhof @gabrielschulhof (TSC)
+* Gireesh Punathil @gireeshpunathil (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Michaël Zasso @targos (TSC)
+* Tobias Nießen @tniessen (TSC)
+* Rich Trott @Trott (TSC)
+
+## Agenda
+
+### Announcements
+
+* Gabriel - new webidl-NAPi project to translate webidl to NAPI code
+* Michael - 14.x went LTS this week !
+
+### CPC and Board Meeting Updates
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* Board meeting this week. Nothing new that I have to bring up with them.
+
+### nodejs/node
+
+* [34895](https://github.com/nodejs/node/pull/34895), blocked on on TSC approvals.
+ * new top module -> diagnostics_channel
+ * @Quard owns namespace in npm, so no chance of collision
+ * Needs 2 TSC approvals
+   * No objections/concerns raised in the meeting.
+
+* Backporting `util/types`, `path/posix` and `path/win32` to v15 proposal [#35788](https://github.com/nodejs/node/issues/35788)
+  * Myles, concern not with this change, but policy around new named modules, /xxx promises
+    is getting in the way. We should find some sort of pattern, or be ok with more ecosystem 
+    Breakage.
+  * Rich past discussion, but no agreement on what the namespace should be. 
+  * Myles, have started using nodejs: loader internally. Can be used with ES Modules, 
```

Comment:
The scheme is `node:`, not `nodejs:` (see <https://github.com/nodejs/node/pull/35387>)
```suggestion
  * Myles, have started using node: loader internally. Can be used with ES Modules, 
```